models/curvefi/ethereum/curvefi_ethereum_view_pools.sql

```diff
@@ -292,6 +260,8 @@ SELECT
     undercoin1,
     undercoin2,
     undercoin3,
+    array(undercoin0, undercoin1, undercoin2, undercoin3) as undercoins,
+    array(coin0, coin1, coin2, coin3) as coins, --changing order to hopefully reset the CI
```

Comment:
do you want to move this back now? how much does order matter?
in general, editing a model is a good way to get it to trigger via CI, so good call there. just want to make sure you don't need the order back. you can also just add a comment or spacing to get it triggered